src/scss/layouts/WordsOverview.scss

```diff
@@ -8,28 +6,71 @@
 
     .main_container--grid {
         margin: 30px;
+        margin-top: 0;
+        min-width: 80%;
     }
 
-        .wordBorder {
-            display: grid;
-            grid-template: 100% / repeat(3, 1fr) 1fr;
-            border: 2px solid rgb(168, 166, 163);
+    .wordBorder {
+        display: grid;
+        grid-template-columns: 1.5fr 0.5fr 1.2fr 1fr;
+        grid-template-rows: 29px;
+        border: 2px solid rgb(168, 166, 163);
+        margin: 10px;
+        padding-left: 20px;
+        box-shadow: $box-shadow;
+        background-color: $container-color;
+        height: 30px;
+
+        .transWorsDe,
+        .transWordCz {
+            border: 1px solid rgb(168, 166, 163);
+            width: auto;
+            overflow: hidden;
+            white-space: nowrap;
+            text-overflow: ellipsis;
+        }
+
+        p.transWorsDe,
+        p.transWordCz {
+            color: #262548;
+            font-size: 14px;
+            font-weight: 600;
+            text-align: center;
+            margin: 5px;
+            padding: 1px;
+            align-self: center;
+        }
+
+        .trans {
```

Comment:
Když předěláš struktroru transWorsDe a transWorsCz tak zvaž jestli předělat i trans paragraph, uvidíš jak se to bude jako celek chovat.

Reply:
Předěláno